SQL/updates/34-35.sql

```diff
@@ -0,0 +1,4 @@
+# Updates DB from 34 to 35 -hal9000PR
+# Changes DB default for clientFPS to 63
+
+ALTER TABLE `player` CHANGE COLUMN `clientfps` `clientfps` INT(11) DEFAULT '63' AFTER `exp`;
```

Comment:
The schema is set for a `smallint(4)`, but here you have `int(11)`. Intended?

Also side note, do we need the `AFTER col_name` syntax here? The query appears to works fine without it. I noticed the update queries we have do tend to use this though. This is more something for Affected can answer.

Reply:
`Also side note, do we need the AFTER col_name syntax here? The query appears to works fine without it. I noticed the update queries we have do tend to use this though. This is more something for Affected to answer.`

I prefer having it like this for maximum sanity. And yes it should be `smallint(4)`.